glusterd2/pmap/registry.go

```diff
@@ -139,18 +163,41 @@ func (r *pmapRegistry) Remove(port int, brickpath string, conn net.Conn) error {
 	return nil
 }
 
+func (r *pmapRegistry) reconcileFirewalld() {
+	sigCh := make(chan *dbus.Signal, 10)
```

Comment:
can you  add a comment about why  channel  length is taken as 10

Reply:
It is arbitrary. The dbus API requires it to be buffered.

From:
https://godoc.org/github.com/godbus/dbus#Conn.Signal

"The caller has to make sure that ch is sufficiently buffered; if a message arrives when a write to c is not possible, it is discarded."